kalitectl.py

```diff
@@ -571,6 +572,27 @@ def start(debug=False, watch=False, daemonize=True, args=[], skip_job_scheduler=
             'server.socket_port': port
         })
 
+    # idealy we'd have a setting element, but these are imported much later
+    # and it will require a much deeper change
+    idle_timeout = int(os.getenv("KALITE_IDLE_TIMEOUT", 0))
```

Comment:
This does not look correct. If the environment variable does not exist, getenv will return the zero value you specified and therefore the logic in the if block below will always be executed... with a zero timeout configured, meaning that the server will be stopped right away.

I think a better way to do it is not to specified the zero value in getenv and compare whether the output of getenv is `None` before parsing its value into a number and executing the logic below.